packages/common/src/utils/dependencies.ts

```diff
@@ -33,28 +33,26 @@ async function fetchAllVersions(dep: string): Promise<JsDelivrApiResult> {
   );
 }
 
+/** Resolves version range from unpkg, use this as a fallback when jsdelivr fails */
+const resolveVersionFromUnpkg = (
+  dep: string,
+  version: string
+): Promise<string> => {
+  return fetchWithRetries(
+    `https://unpkg.com/${dep}@${encodeURIComponent(version)}/package.json`
+  ).then(x => x.version);
+};
+
 export async function getLatestVersion(
```

Comment:
I triple checked, I think this function is nowhere else used. We could remove the `export` here.
```suggestion
async function getLatestVersion(
```